HADOOP-11212. NetUtils.wrapException to handle SocketException explicitly. (Contributed by Steve Loughran)

+    } else if (exception instanceof SocketException) {
+      // Many of the predecessor exceptions are subclasses of SocketException,
+      // so must be handled before this
+      return wrapWithMessage(exception,
+          "Call From "
+              + localHost + " to " + destHost + ":" + destPort
+              + " failed on socket exception: " + exception
+              + ";"
+              + see("SocketException"));
-                                               + exception
-                                               + "; Host Details : "
-                                               + getHostDetailsAsString(destHost, destPort, localHost))
+             + exception
+             + "; Host Details : "
+             + getHostDetailsAsString(destHost, destPort, localHost))

INS25 INS62 INS8 MOV8 INS42 INS43 INS41 INS42 INS32 INS42 INS42 INS27 INS45 INS42 INS45 INS42 INS45 INS42 INS45 INS42 INS45 INS32 INS42 INS45